Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -4,6 +4,8 @@
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
 from Configuration.Eras.Modifier_trackingPhase1_cff import trackingPhase1
 from Configuration.Eras.Modifier_run2_HE_2017_cff import run2_HE_2017
+from Configuration.Eras.Modifier_run2_HF_2017_cff import run2_HF_2017
+from Configuration.Eras.Modifier_run2_HCAL_2017_cff import run2_HCAL_2017
 
-Run2_2017 = cms.ModifierChain(Run2_2016, phase1Pixel, trackingPhase1, run2_HE_2017)
+Run2_2017 = cms.ModifierChain(Run2_2016, phase1Pixel, trackingPhase1, run2_HE_2017, run2_HF_2017, run2_HCAL_2017)
```

Comment:
@makortel yes, I thought these were already synced automatically, but if not, let's fix it